source/game/kart/KartMove.hh

```diff
@@ -234,7 +244,7 @@ public:
     void onWallCollision() override;
     void calcMtCharge() override;
     void setTurnParams() override;
-    void init(bool b1, bool b2) override;
+    void init(bool keepStatsForCannon, bool b2) override;
```

Comment:
If you're going to rename it in the header, please rename it in the source file too

Reply:
ill just revert that since vabold obeserved that ghidra subtly hides the fact that actualy *both* bools are set. and i cant come up with good names for both of them yet.